### PR TITLE
Forgetpassword fixes

### PR DESCRIPTION
**Pull request by asma-0408**

### CODE REVIEW

**Comment by asma-0408** [comment]
The tests are failing but I think there are issues with the cart apis. Could you take a look and confirm?

**Comment by Timtaoz** [comment]
I'll go check it

File: fleaapi/helloworld/__init__.py
Comment (tpcaibu) [inline fleaapi/helloworld/__init__.py:1]: Thanks for letting me know!
Revert this line solves your problem, but I have another test fail which is related to your change.

```
self = <fleaapi.controllers.unittest.forget_test.PasswordResetTests testMethod=test_reset_password>

    def test_reset_password(self):
        # Test successful password change
        response = self.client.post(reverse('reset_password'), {'email': 'test@example.com', 'new_password': 'new_password'})
>       self.assertEqual(response.status_code, 200)
E       AssertionError: 404 != 200

fleaapi/controllers/unittest/forget_test.py:24: AssertionError
```

**Comment by tpcaibu** [comment]
I think the tests are working now, but the coverage seems low. Either removing unused code or writing more tests can increase the test coverage.

**Pull request merged**